CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+## The Mantra
+
+We are great believers in open source and the great community that it builds, we feel that is anyone's right to participate in the development of Grüvee, this means everyone should have equal oppertunity to contrubute to the awesome world of open source, and at the same time be treated with the highest levels of respect.
+
+At the end of the day we're all here to build something awesome, so let's go ahead and make something awesome!
+
```

Comment:
This is cute as heck. 😻 🖤

Reply:
Glad it's okay! I really wanted something to just lay down the expectations of contributing  but didn't want it sound too formal.